File: unit11-hw/ruby/WalkingBird.rb
# WalkingBird.rb | Defines the WalkingBird class which inherits attributes and
#   methods from the Bird superclass.
#
# Completed by: Robert Lowman
# Date: 12/5/16
####################################################

require_relative 'Bird.rb'

class WalkingBird < Bird

  def movement
    'walked past'
  end

end

File: unit11-hw/python/WalkingBird.py
# WalkingBird.py | Defines the WalkingBird class which inherits attributes and
#   methods from the Bird superclass.
#
# Completed by: Robert Lowman
# Date: 12/7/16
####################################################

import Bird

class WalkingBird(Bird.Bird):

    def __init__(self, name):
        super().__init__(name)

    def movement(self):
        return 'walked past'
